src/palette/internal/palettecreator.cpp

```diff
@@ -1104,14 +1104,23 @@ PalettePtr PaletteCreator::newGraceNotePalette()
     sp->setDrawGrid(true);
     sp->setVisible(false);
 
-    sp->appendActionIcon(ActionIconType::ACCIACCATURA,  "acciaccatura");
-    sp->appendActionIcon(ActionIconType::APPOGGIATURA,  "appoggiatura");
-    sp->appendActionIcon(ActionIconType::GRACE4,        "grace4");
-    sp->appendActionIcon(ActionIconType::GRACE16,       "grace16");
-    sp->appendActionIcon(ActionIconType::GRACE32,       "grace32");
-    sp->appendActionIcon(ActionIconType::GRACE8_AFTER,  "grace8after");
-    sp->appendActionIcon(ActionIconType::GRACE16_AFTER, "grace16after");
-    sp->appendActionIcon(ActionIconType::GRACE32_AFTER, "grace32after");
+    sp->appendActionIcon(ActionIconType::ACCIACCATURA,  QT_TRANSLATE_NOOP("engraving", "Acciaccatura"));
+    sp->appendActionIcon(ActionIconType::APPOGGIATURA,  QT_TRANSLATE_NOOP("engraving", "Appoggiatura"));
+#if 1 // taken from note.cpp (like the above), no additional translations needed
```

Comment:
Not sure which mwethood to use, comments wanted...

Reply:
second? so that the names are not repeated

Reply:
But note.cpp doesn't make the distinction. Although maybe it should?

Reply:
Check the next commit...

Reply:
Nah, doesn't work in the status bar, duplicates the duration there

Reply:
@bkunda FYI